### PR TITLE
Mkdocs

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+      - run: pip install -r requirements.txt
+      - run: mkdocs gh-deploy --force
```
Comment (dpsmith) [inline .github/workflows/deploy.yml:16]: Two things, do we want to be deploying on push? Second from what I read in mkdocs, since this is for an organization, you will want `--remote-branch master`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+### Contribution Licensing
+
+TrenchBoot is a cross-community integration project consisting of project
+repositories along with upstream project repositories. All contributions to a
+repository are made under the license that covers all work in that repository.
+
+### Issue Tracking
+
+Report issues, e.g. bugs, feature requests, etc., using Github's
+[issues](https://github.com/briandk/transcriptase-atom/issues)
```
Comment (dpsmith) [inline CONTRIBUTING.md:78]: Is this URL a copy artifact?

Comment (dpsmith) [reply CONTRIBUTING.md:78]: oh, was this what was already in there??

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+[Creative Commons Attribution 4.0 International
+License](ttp://creativecommons.org/licenses/by/4.0/).  By contributing you
+agree that your contributions will be licensed under the Creative Commons
+Attribution 4.0 International License. Feel free to contact the maintainers if
+that's a concern.
+
+## Contacting Maintainers
+
+TrenchBoot is maintained by:
+ - Daniel P. Smith <dpsmith@apertussolutions.com>
```
Comment (dpsmith) [inline CONTRIBUTING.md:116]: Probably should reword this to be,

TrenchBoot is maintained by TrenchBoot Steering Committee, current members:
Daniel P. Smith
Ross Philipson
Piotr Krol
Rich Persaud

**Review by dpsmith — APPROVED** [review]
I started to review and realized there is probably a bit we should get updated and we really should get this published.

File: CONTRIBUTING.md (hunk 2 above)
Comment (miczyg1) [reply CONTRIBUTING.md:78]: Yes, this is the old contents from TrenchBoot/documentation repo: https://github.com/TrenchBoot/documentation/blob/master/CONTRIBUTING.md?plain=1#L78
